morpheus/stages/inference/inference_stage.py

```diff
@@ -230,9 +230,9 @@ def on_next(x: MultiInferenceMessage):
                 for batch in batches:
                     outstanding_requests += 1
 
-                    fut = neo.Future()
+                    fut = srf.Future()
 
-                    def set_output_fut(resp: ResponseMemoryProbs, b, f: neo.Future):
+                    def set_output_fut(resp: ResponseMemoryProbs, b, f: srf.Future):
```

Comment:
nit pick: can we shift toward more informative parameter names?

Reply:
Done. Went with `batch_future`